src/arch/x86_64/print_mir.zig

```diff
@@ -0,0 +1,580 @@
+//! This file contains the functionality for lowering x86_64 MIR into
```

Comment:
Nitpick: As this file follows the struct-as-file pattern, I think it would make sense to adjust the file name to reflect that (e.g. `Print.zig` or `PrintMir.zig`)